modules/apps/accessibility/accessibility-menu-web/src/main/resources/META-INF/resources/js/util.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {localStorage} from 'frontend-js-web';
+
+export const SETTINGS_STRING_VALUES_MAP = {
+	false: false,
+	true: true,
+	undefined,
+};
+
+export function getSettingValue(defaultValue, sessionClicksValue, key) {
+	if (
+		!themeDisplay.isSignedIn() ||
+		SETTINGS_STRING_VALUES_MAP[sessionClicksValue] === undefined
```

Comment:
Wouldn't it be better to just check `sessionClicksValue === undefined` or `typeof sessionClicksValue === 'undefined'`? What would be the purpose of `SETTINGS_STRING_VALUES_MAP`?

Reply:
Yep, I was able to rework this to get rid of SETTINGS_STRING_VALUES_MAP. Initially I found it complex because sessionClicks only returns a string, but I'm able to process that on the backend and make it simpler on the frontend for us. Thanks.